backend/routes/doctorOrders.js

```diff
@@ -8,10 +8,10 @@ const axios = require('axios').default;
 
 const DoctorOrder = require('../models/doctorOrders');
 
-router.patch("/fhir/rems/:id", async (req, res, next) => {
+router.patch("/$process-message/:id", async (req, res, next) => {
   const documentId = req.params.id;
   let documentOrder = await DoctorOrder.findById(documentId);
-  const crdUrl = (process.env.CRD_BASE_URL ? process.env.CRD_BASE_URL : "http://localhost:8090/") + "rems/" + documentOrder.caseNumber;
+  const crdUrl = (process.env.CRD_BASE_URL ? process.env.CRD_BASE_URL : "http://localhost:8090/") + "api/rems/" + documentOrder.caseNumber;
```

Comment:
I see, I'm not aware of which endpoints are used since I'm new to the team and I haven't been able to test out implementation with postman. I can check in with Sahil about this and make a new request if necessary